lib/naclCrypto.js

```diff
@@ -0,0 +1,11 @@
+import nacl from 'tweetnacl';
```

Comment:
If we're not using https://www.npmjs.com/package/@stablelib/x25519 in this PR, could you file an issue to switch to it?

Reply:
@dlongley sure. If you'd like, I can take a stab at switching over to stablelib, in this PR?

Reply:
@dmitrizagidulin, sure! Do you mind also figuring out what the size difference would be (roughly) if we switched?

Reply:
Looks like tweetnacl is only ~18k minimized. Just need to know what stabelib/x25519 would be.

Reply:
@dlongley got it -- this should now be captured in issue https://github.com/digitalbazaar/x25519-key-agreement-key-2019/issues/13